src/daemon.py

```diff
@@ -165,32 +167,34 @@ def write_token_imbalances_to_db(
         )
 
 
-def write_coingecko_prices(
+def write_prices(
     chain_name: str,
```

Comment:
The PR is not about that but let me just throw it in here anyways. Create an issue out of it if you do not want to address it here.

The convention in the services repo is to use `network` for the name of the chain. There is also an environment variable `NETWORK` set up containing the chain.
The names for chains currently used are `mainnet`, `xdai`, and `arbitrum-one`.
We should only deviate from that convention if we have a really good reason to do that.